### PR TITLE
Added creation sounds

### PR DESCRIPTION
**Pull request by Dzierzan**
- Added 2 new creation sounds, one for base building, the other for actor creation
- Simplified buildings.yaml by moving traits to default.yaml
- Made base turrets and super weapons uncapturable

### CODE REVIEW

File: mods/hv/rules/buildings.yaml [hunk 1]
```diff
@@ -1069,13 +1009,10 @@ SILO:
...
-		Condition: build-incomplete
-	WithConstructionBeamOverlay:
-		RequiresCondition: !build-incomplete
-		Condition: beam-incomplete
-	Capturable:
-		RequiresCondition: !build-incomplete && !beam-incomplete
+	-Capturable:
+	-CapturableProgressBar:
+	-CapturableProgressBlink:
+	-CaptureNotification:
```
Comment (Mailaender) [inline mods/hv/rules/buildings.yaml:1015]: These change look unrelated. Why should the missile silo not be capturable?

Comment (Dzierzan) [reply mods/hv/rules/buildings.yaml:1015]: Super weapons are not capturable in CnC games.

Comment (Mailaender) [reply mods/hv/rules/buildings.yaml:1015]: It does not make sense to me and I don't like it.

File: OpenRA.Mods.HV/Traits/Render/WithProductionExitOverlay.cs [hunk 2]
```diff
@@ -35,6 +35,9 @@ public class WithProductionExitOverlayInfo : ConditionalTraitInfo, Requires<Rend
 		[Desc("Custom palette is a player palette BaseName.")]
 		public readonly bool IsPlayerPalette = false;
 
+		[Desc("Plays a sound when an actor is spawned.")]
+		public readonly string[] CreationSound = { };
```
Comment (Mailaender) [inline OpenRA.Mods.HV/Traits/Render/WithProductionExitOverlay.cs:39]: You are adding sound to a render trait which is not the proper way.

**Comment by Dzierzan** [comment]
Alright, I'll close it and seperate that into two PRs.
